### ISSUE TITLE
font-size mixin

### ISSUE THREAD

**Issue by ptbrowne** [cozy/cozy-ui#258]
`font-size` is redefined here : https://github.com/cozy/cozy-ui/blob/5e092f964de2c2268a004250bac5eab019a5d5a0/stylus/tools/mixins.styl

It basically convert the px in em so that your px are dependent on the root font size. The primary goal dixit m4dz was to help developers that do not know about `rem` to have a design that was responsive and reactive to the change of root font size.

I think it is a very surprising behavior of cozy-ui, that `rem` fit this goal perfectly, and it does not teach people how to use properly `rem`. 

I would be in favor of removing this mixin.

**Comment by y-lohse** [comment]
Yup, same here. I think we've had some accidents with this mixin without realizing we were using it.

**Comment by GoOz** [comment]
Well is was useful when we used `em` but yeah I think its traitorous the way it is.
I'm kinda in favour to remove it **BUT** It may breaks some font-size here and there because it's still being used. And I guess it's used a lot.

Do you have any advice on the best way to deal with it?

**Comment by ptbrowne** [comment]
I think the best way to deal with this change is to do it gradually and to provide everyone with the right information to deal with this change in their code themselves. I added in https://github.com/cozy/cozy-ui/pull/260 a way to display deprecation messages behind a flag and I used this feature to provide info on how to deal with the future disparition (I hope) of font-size and em. Let me know what you think about this.
